Converter.cs

```diff
@@ -789,17 +789,17 @@ public static Tie.Mission XwingToTie(Xwing.Mission miss)
                             case 2: cond = 12; amount = 0; break; //100% complete mission
                             case 3: cond = 4; amount = 0; break; //100% captured
                             case 4: cond = 6; amount = 0; break; //100% be boarded
-                            case 5: cond = 2; amount = 4; break; //special craft destroyed
-                            case 6: cond = 12; amount = 4; break; //special craft complete mission
-                            case 7: cond = 4; amount = 4; break; //special craft captured
-                            case 8: cond = 6; amount = 4; break; //special craft be boarded
-                            case 9: cond = 2; amount = 1; break; //50% destroyed
-                            case 10: cond = 12; amount = 1; break; //50% complete mission
-                            case 11: cond = 4; amount = 1; break; //50% captured
-                            case 12: cond = 6; amount = 1; break; //50% be boarded
+                            case 5: cond = 2; amount = 6; break; //special craft destroyed
```

Comment:
TIE is actually correct, since FG Goals use a different enum than normal Triggers.  I'll revert after the merge.